scripts/setup_env.ps1

```diff
@@ -0,0 +1,67 @@
+$current_dir = Get-Location
```

Comment:
Shouldn't we rename `scripts` to `tools` and `setup_env` to `manage` to match all other repositories?

Reply:
well, I am not a friend of `manage` script because it is hiding functionality. Also any changes in one place of the script might affect the rest of it? `scirpts` -> `tools` I can live with, but mange script I would avoid. Even the difference between windows platform "manage" and linux "make" we have is bothering me...

Reply:
> Also any changes in one place of the script might affect the rest of it?

Well but you can change it in single file, which can have help to provide more information than just filename. Also helps to avoid duplicated code, that has to be changed in every separate script if you need to change it.